pallets/operations/src/lib.rs

```diff
@@ -53,22 +56,29 @@ pub mod pallet {
     #[pallet::event]
     #[pallet::generate_deposit(pub(super) fn deposit_event)]
     pub enum Event<T: Config> {
-        /// An account has fixed its consumers counter underflow
-        ConsumersUnderflowFixed { who: T::AccountId },
+        /// A consumers counter was incremented for an account
+        ConsumersCounterIncremented { who: T::AccountId },
+
+        /// A consumers counter was decremented for an account
+        ConsumersCounterDecremented { who: T::AccountId },
     }
 
     #[pallet::call]
     impl<T: Config> Pallet<T> {
-        /// An account can have an underflow of a `consumers` counter.
-        /// Account categories that are impacted by this issue depends on a chain runtime,
-        /// but specifically for AlephNode runtime are as follows:
+        /// An account can have an underflow or overflow of a `consumers` counter.
+        /// Expected consumers counter depends on a chain runtime,
+        /// but specifically for AlephNode runtime is as follows:
         /// +1 consumers if reserved > 0 || frozen > 0
-        /// +1 consumers if there is at least one lock (staking or vesting)
-        /// +1 consumers if there's session.nextKeys set, for controller account
+        /// +1 if the account is a contract one
+        /// +1 consumers for a controller account that is in session.next_keys
         /// +1 consumers if account bonded
         ///
-        ///	`fix_accounts_consumers_underflow` calculates expected consumers counter and comperes
+        ///	`fix_accounts_consumers_counter` calculates expected consumers counter and compares
         /// it with current consumers counter, incrementing by one in case of an underflow
+        /// or decrementing it by one in case of an overflow
+        ///
+        /// When expected is different from current by more than one, you might want to
+        /// call this extrinsic more than once.
```

Comment:
Actually why wont we fix it in one call?

Reply:
I don't want to do code like this 
```
while expected_counter < current counter {
   inc_consumers();
}
```
just in case there would be some unforeseen changes in pallet balances in the future that might cause happen the loop is not to work as expected, e.g., infinite loop. I know it's almost not possible, yet I want to be on the safe side when placing a loop in the runtime code.

Reply:
I don't understand the reasoning. We will just remove this call in the future version, right?